controller/revert.go

```diff
@@ -61,12 +61,16 @@ func (c *Controller) Revert(name string) error {
 	}
 
 	if err := c.startFrontend(); err != nil {
-		return err
+		if !strings.Contains(err.Error(), "no frontend started since it's not specified") {
```

Comment:
If the frontend wasn't enabled, then we shouldn't call `shutdown` and `start` frontend.